scripts/byobu_startup.py

```diff
@@ -172,37 +209,37 @@ def setup_main_control_window(cmd_prefix='#'):
     # setup the main-control window
     select_window(WINDOWS[0])
     # Select default pane. Probably an unnecessary line of code
-    subprocess.call(f"byobu select-pane -t 0", shell=True)
+    call_byobu(f"select-pane -t 0")
     # split window hoirzontaly
-    subprocess.call(f"byobu split-window -h", shell=True)
+    call_byobu(f"split-window -h")
     # select pane 0
-    subprocess.call(f"byobu select-pane -t 0", shell=True)
+    call_byobu(f"select-pane -t 0")
     # split selected pane vertically
-    subprocess.call(f"byobu split-window -v", shell=True)
+    call_byobu(f"split-window -v")
     # select the top pane
-    subprocess.call(f"byobu select-pane -t 0", shell=True)
+    call_byobu(f"select-pane -t 0")
     # split top pane vertically again
-    subprocess.call(f"byobu split-window -v", shell=True)
+    call_byobu(f"split-window -v")
     # select the top pane
-    subprocess.call(f"byobu select-pane -t 0", shell=True)
+    call_byobu(f"select-pane -t 0")
 
     clear_current_pane()
     # Now run the necessary commands in each pane
     send_command_to_pane(
-        cmd_prefix + '$HUNTSMAN_POCS/scripts/pyro_name_server.py', 0)
+        cmd_prefix + '${HUNTSMAN_POCS}/scripts/pyro_name_server.py', 0)
 
     send_command_to_pane(
-        cmd_prefix + 'python $HUNTSMAN_POCS/scripts/start_config_server.py', 1)
+        cmd_prefix + 'python ${HUNTSMAN_POCS}/scripts/start_config_server.py', 1)
 
-    send_command_to_pane(f'ipython', 2)
+    send_command_to_pane(f'jupyter-lab', 2)
 
-    for cmd in POCS_STARTUP:
-        send_command_to_pane(cmd_prefix + cmd, 2)
-        # issues occur without small pause between commands, unsure why
-        time.sleep(0.01)
+    #for cmd in POCS_STARTUP:
+    #    send_command_to_pane(cmd_prefix + cmd, 2)
+    #    # issues occur without small pause between commands, unsure why
+    #    time.sleep(0.01)
 
-    send_command_to_pane(
-        cmd_prefix + 'grc tail -F -n 1000 $PANDIR/logs/ipython-all.log', 3)
+    #send_command_to_pane(
+    #    cmd_prefix + 'grc tail -F -n 1000 $PANDIR/logs/ipython-all.log', 3)
```

Comment:
If this is going to stay commented let's remove. Otherwise shouldn't the "dry run" flag make it so that it skips this?

Reply:
I think this was something that was changed on the control computer and I just copied it over. I'm not sure why someone chose to comment it out